Lib/multiprocessing/context.py

```diff
@@ -199,15 +202,28 @@ def get_start_method(self, allow_none=False):
     def set_start_method(self, method, force=False):
         raise ValueError('cannot set start method of concrete context')
 
-    @property
-    def reducer(self):
+    def get_reducer(self):
         '''Controls how objects will be reduced to a form that can be
         shared with other processes.'''
-        return globals().get('reduction')
+        reducer = getattr(self, "_reducer", None)
+        if reducer is None:
+            return reduction
+        return reducer
+
+    def set_reducer(self, reduction):
+        if reduction is original_reducer:
+            self._reducer = None
+            return
+
+        if not isinstance(reduction, original_reducer.AbstractReducer):
+            raise TypeError("Custom reducers must be instances of a "
+                            "subclass of multiprocessing.reduction.AbstractReducer")
+        if not isinstance(reduction.get_pickler(), pickle.Pickler):
+            breakpoint()
```

Comment:
You want to remove this :-)

Reply:
Whoops! Sorry, I finished this very late at night  :)